lms/templates/instructor/instructor_dashboard_2/open_response_assessment.html

```diff
@@ -1,6 +1,8 @@
 <%page args="section_data" expression_filter="h"/>
 <%! from openedx.core.djangolib.markup import HTML %>
 
+<%include file="/courseware/xqa_interface.html/"/>
+
```

Comment:
Convert the link to `static` link. We won't have to worry about the file inclusion in different environments then.

Reply:
Ignore this. Not needed